src/Microsoft.Tye.Hosting/ProcessRunner.cs

```diff
@@ -347,6 +347,12 @@ private void WriteReplicaToStore(string pid)
             return null;
         }
 
+        private static string GetEntryPointFilePath()
+        {
+            using var process = Process.GetCurrentProcess();
```

Comment:
This doesn't seem right.

Reply:
Does this return a new instance every time?

Reply:
yep

```
public static Process GetCurrentProcess()
{
	return new Process(".", isRemoteMachine: false, NativeMethods.GetCurrentProcessId(), null);
}
```

Reply:
i think "returns new instance" is meant to be implied by the fact that it is a method and not a property